lib/iris/util.py

```diff
@@ -471,7 +472,7 @@ def reverse(cube_or_array, coords_or_dims):
                                 'or sequence of these.')
 
     axes = np.array(list(axes), ndmin=1)
-    if axes.ndim != 1:
+    if axes.ndim != 1 or axes.size == 0:
```

Comment:
This is added because the tests that pass an empty string were failing at `np.min` (line 478 below) and the error message was less informative in context.